chain/chain/src/chain.rs

```diff
@@ -3199,10 +3199,6 @@ impl<'a> ChainUpdate<'a> {
             return Err(ErrorKind::InvalidChunkMask.into());
         }
 
-        if !header.verify_chunks_included() {
-            return Err(ErrorKind::InvalidChunkMask.into());
```

Comment:
why is this removed?

Reply:
I wrongly assumed this check is not necessary because it's always true. Reverted.